gui/default/assets/lang/lang-en.json

```diff
@@ -416,9 +439,11 @@
     "The GUI address is overridden by startup options. Changes here will not take effect while the override is in place.": "The GUI address is overridden by startup options. Changes here will not take effect while the override is in place.",
     "The Syncthing Authors": "The Syncthing Authors",
     "The Syncthing admin interface is configured to allow remote access without a password.": "The Syncthing admin interface is configured to allow remote access without a password.",
+    "The Syncthing admin interface is configured to allow remote access without authentication.": "The Syncthing admin interface is configured to allow remote access without authentication.",
     "The aggregated statistics are publicly available at the URL below.": "The aggregated statistics are publicly available at the URL below.",
     "The cleanup interval cannot be blank.": "The cleanup interval cannot be blank.",
     "The configuration has been saved but not activated. Syncthing must restart to activate the new configuration.": "The configuration has been saved but not activated. Syncthing must restart to activate the new configuration.",
+    "The credentials below were created with a different setting of \"Webauthn Rp Id\" in advanced settings.\n                        They cannot be used without restoring this setting to the value shown in the \"RP ID\" column below.\n                        Note that doing so would make any credentials above ineligible instead.": "The credentials below were created with a different setting of \"Webauthn Rp Id\" in advanced settings.\n                        They cannot be used without restoring this setting to the value shown in the \"RP ID\" column below.\n                        Note that doing so would make any credentials above ineligible instead.",
```

Comment:
This shows how non-trimmed whitespace can really mess up translation strings.  You should either put each sentence into its own translatable paragraph, or just not wrap the paragraph into multiple source code lines.  I'd prefer the latter in this case.

Reply:
Fixed in 974549c21f0a5e1b36b6ed3d20466cbdb0c463c8.